com.unity.cinemachine/Editor/Editors/CinemachineConfiner3DEditor.cs

```diff
@@ -0,0 +1,102 @@
+#if CINEMACHINE_PHYSICS 
+
+using UnityEngine;
+using UnityEditor;
+using UnityEditor.UIElements;
+using UnityEngine.UIElements;
+
+namespace Cinemachine.Editor
+{
+    [CustomEditor(typeof(CinemachineConfiner3D))]
+    [CanEditMultipleObjects]
+    class CinemachineConfiner3DEditor : UnityEditor.Editor
+    {
+        CinemachineConfiner3D Target => target as CinemachineConfiner3D;
+
+        CmPipelineComponentInspectorUtility m_PipelineUtility;
+
+        void OnEnable() => m_PipelineUtility = new (this);
+        void OnDisable() => m_PipelineUtility.OnDisable();
+
+        public override VisualElement CreateInspectorGUI()
+        {
+            var ux = new VisualElement();
+
+            m_PipelineUtility.AddMissingCmCameraHelpBox(ux);
+
+            var boundsHelp = ux.AddChild(new HelpBox(
+                "Bounding Volume must be a BoxCollider, SphereCollider, CapsuleCollider, or convex MeshCollider.", 
+                HelpBoxMessageType.Warning));
+
+            var volumeProp = serializedObject.FindProperty(() => Target.BoundingVolume);
+            ux.Add(new PropertyField(volumeProp));
+            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.SlowingDistance)));
+            m_PipelineUtility.UpdateState();
+
+            TrackVolume(volumeProp);
+            ux.TrackPropertyValue(volumeProp, TrackVolume);
+            void TrackVolume(SerializedProperty p)
+            {
+                var c = p.objectReferenceValue;
+                var mesh = c as MeshCollider;
+                var isConvexMesh = mesh != null && mesh.convex;
+                boundsHelp.SetVisible(!(isConvexMesh || c is BoxCollider || c is SphereCollider || c is CapsuleCollider));
+            }
+            return ux;
+        }
+
+        [DrawGizmo(GizmoType.Active | GizmoType.Selected, typeof(CinemachineConfiner3D))]
+        private static void DrawColliderGizmos(CinemachineConfiner3D confiner, GizmoType type)
+        {
+            CinemachineVirtualCameraBase vcam = (confiner != null) ? confiner.VirtualCamera : null;
+            if (vcam != null && confiner.IsValid)
+            {
+                var oldMatrix = Gizmos.matrix;
+                var oldColor = Gizmos.color;
+                Gizmos.color = Color.yellow;
+
+                var t = confiner.BoundingVolume.transform;
+                Gizmos.matrix = Matrix4x4.TRS(t.position, t.rotation, t.lossyScale);
+
+                var colliderType = confiner.BoundingVolume.GetType();
+                if (colliderType == typeof(BoxCollider))
+                {
+                    var c = confiner.BoundingVolume as BoxCollider;
+                    Gizmos.DrawWireCube(c.center, c.size);
+                }
```

Comment:
I added this under _line 65_ to visualize when the slow happens.
```
Gizmos.color = Color.red;
Gizmos.DrawWireCube(c.center, c.size - 4f * new Vector3(confiner.SlowingDistance / t.lossyScale.x, 
                                                                            confiner.SlowingDistance / t.lossyScale.y, 
                                                                            confiner.SlowingDistance / t.lossyScale.z));
```

Reply:
The slowing can be controlled by increasing the distance param.  Set it very high, then you can see that the vcam begins to slow much sooner.  We can try to think of a setting that will change the shape of the slowing curve, but it is tricky.  Right now the curve looks like this, so it's hard to see any visible effect except near the end 
![image](https://user-images.githubusercontent.com/6424658/205321712-8e273387-4de6-40b8-86c9-f7b220cc5459.png)